go/engine/passphrase_change.go

```diff
@@ -116,27 +123,26 @@ func (c *PassphraseChange) findPaperKeys(m libkb.MetaContext) (*libkb.DeviceWith
 // The first choice is device keys.  If that fails, it will look
 // for backup keys.  If backup keys are necessary, then it will
 // also log the user in with the backup keys.
-func (c *PassphraseChange) findUpdateKeys(m libkb.MetaContext) (*libkb.DeviceWithKeys, error) {
-	kp, err := c.findDeviceKeys(m)
-	if err == nil {
-		return kp, nil
+func (c *PassphraseChange) findUpdateKeys(m libkb.MetaContext) (ad *libkb.ActiveDevice, err error) {
```

Comment:
it's a little confusing to me that this returns an ActiveDevice object.  maybe `findActiveDevice`?  since below when you call it you do `ad, err := findUpdateKeys(...)`

Reply:
sure thing